activerecord/lib/active_record/migration.rb

```diff
@@ -1047,14 +1047,34 @@ def load_migration
       end
   end
 
-  class MigrationContext #:nodoc:
+  # MigrationContext sets the context in which a migration is run.
+  #
+  # A migration context requires the path to the migrations is set
+  # in the `migrations_paths` parameter. Optionally a `schema_migration`
+  # class can be provided. For most applications, `SchemaMigration` is
+  # sufficient. Multiple database applications need a `SchemaMigration`
+  # per primary database.
+  class MigrationContext
     attr_reader :migrations_paths, :schema_migration
 
-    def initialize(migrations_paths, schema_migration)
+    def initialize(migrations_paths, schema_migration = SchemaMigration)
       @migrations_paths = migrations_paths
       @schema_migration = schema_migration
     end
 
+    # Runs the migrations in the `migrations_path`.
+    #
+    # If `target_version` is `nil`, `migrate will run `up`.
```

Comment:
@eileencodes Feels like migrate is missing closing quote(`) here. 😊

Reply:
Also use `+` instead of ``` ` ``` in the rdoc.